scripts/sync_protobuf/_helpers.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+import subprocess
+import sys
+from http.client import HTTPResponse
+from pathlib import Path
+from typing import TYPE_CHECKING, Iterable
+from urllib.request import urlopen
```

Comment:
I'm purposefully avoiding requests here, as to not add `requests` and `types-requests` in `requirements-tests.txt`